src/mt2_bisect.h

```diff
@@ -68,8 +77,9 @@ template <typename T>
 static inline void mt2_swap(T *x, T *y);
 
 
+
 /* Global variables */
-static const double mt2_error = -1.0027064548346334;
+static const float mt2_error = -1.38570487f;
```

Comment:
I haven't been able to figure this one out. Representation in hex isn't meaningful, and if I represent as 4 unicode chars then also not too meaningful... Care to provide a hint?! :-D